### PR TITLE
MAP-287 - Shutter page for providers not in course management service

### PR DESCRIPTION
**Pull request by ratheeshri1984**

### CODE REVIEW

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CohortController.cs [hunk 1]
```diff
@@ -277,10 +277,17 @@ public async Task<IActionResult> ConfirmEmployer(ConfirmEmployerRequest request)
         [HttpPost]
         [Route("add/confirm-employer")]
         [Authorize(Policy = nameof(PolicyNames.HasContributorOrAbovePermission))]
-        public IActionResult ConfirmEmployer(ConfirmEmployerViewModel viewModel)
+        public async Task<IActionResult> ConfirmEmployer(ConfirmEmployerViewModel viewModel)
         {
             if (viewModel.Confirm.Value)
             {
+                var model = await _modelMapper.Map<HasDeclaredStandardsViewModel>(viewModel);
```
Comment (chrisfoster76) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CohortController.cs:284]: You might to rename the `HasDeclaredStandardsViewModel` to something that expresses its intent a little more clearly (it's not really a vm). I would go for something like `ConfirmEmployerRedirectModel`, since it's a model that helps us decide where to redirect to from the Confirm Employer post.

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/Review.cshtml [hunk 2]
```diff
@@ -9,8 +9,7 @@
 }
 
 <div class="govuk-grid-row">
-    <div class="govuk-grid-row">
-        <div class="govuk-width-container">
+        <div class="govuk-grid-column-full">
```
Comment (chrisfoster76) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/Review.cshtml:12]: Was this change deliberate?

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/NoDeclaredStandards.cshtml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿@{
+    ViewData["Title"] = "No Declared Standards";
+    ViewBag.GaData.Vpv = "/error";
```
Comment (chrisfoster76) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/NoDeclaredStandards.cshtml:3]: The vpv should be similar to the url for the page.

**Review by chrisfoster76 — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/Review.cshtml (hunk 2 above)
Comment (ratheeshri1984) [reply src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/Review.cshtml:12]: James spotted that in a view yesterday and he fixed it, the html wasn't right.

**Review by chrisfoster76 — COMMENTED** [review]
As previous naming comments for api request/response, otherwise looking good.

**Review by chrisfoster76 — APPROVED** [review]
